Formula/llvm.rb

```diff
@@ -188,17 +226,36 @@ def install
     args = %w[
       -DLLVM_OPTIMIZED_TABLEGEN=On
       -DLLVM_BUILD_LLVM_DYLIB=On
+      -DLLVM_TARGETS_TO_BUILD=X86
```

Comment:
> too many for test runs

Is this just a matter of making them take a long time?

We might want to include `ARM`, too, since that might be a common use case in our audience, with Raspberry Pi hobbyists and similar.

I don't know how commonly the GPU targets would be used, either. Those might be more common under `homebrew/science`. That's just a guess, though.

If we're going to cut down the target set, maybe we should add an option to compile with `all` targets. Otherwise people who need to cross-compile (including CUDA/GPGPU users?) won't be able to use a Homebrew-provided LLVM without hacking the formula, even if they build it from source.